From 451515e288cc42c1522a10d8ab89cf89dcb0f976 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 29 Apr 2021 19:33:34 +0100
Subject: [PATCH 2/2] gnu: Add unit-test

* gnu/packages/lisp-xyz.scm (sbcl-unit-test, ecl-unit-test,
  cl-unit-test):  New variables
---
 gnu/packages/lisp-xyz.scm | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index bd3210d8c1..ecc0582e8d 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -16285,6 +16285,35 @@ color spaces, which supports many color models.")
 (define-public cl-dufy
   (sbcl-package->cl-source-package sbcl-dufy))
 
+(define-public sbcl-unit-test
+  (let ((commit "266afaf4ac091fe0e8803bac2ae72d238144e735")
+        (revision "1"))
+    (package
+      (name "sbcl-unit-test")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/hanshuebner/unit-test")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32
+           "11hpksz56iqkv7jw25p2a8r3n9dj922fyarn16d98589g6hdskj9"))))
+      (build-system asdf-build-system/sbcl)
+      (home-page "https://github.com/hanshuebner/unit-test")
+      (synopsis "Unit-testing framework for common lisp")
+      (description
+"Unit-testing framework for common lisp")
+      (license license:unlicense))))
+
+(define-public ecl-unit-test
+  (sbcl-package->ecl-package sbcl-unit-test))
+
+(define-public cl-unit-test
+  (sbcl-package->cl-source-package sbcl-unit-test))
+
 (define-public sbcl-bknr-datastore
   (let ((commit "c98d44f47cc88d19ff91ca3eefbd9719a8ace022")
         (revision "1"))
-- 
2.31.1

